preferencesfx/src/main/java/com/dlsc/preferencesfx/model/Category.java

```diff
@@ -253,4 +264,8 @@ public ReadOnlyStringProperty descriptionProperty() {
   public Node getItemIcon() {
     return itemIcon;
   }
```

Comment:
I think I would prefer `isExpand` for the getter, to make it obvious

Reply:
Done.